GOVERNANCE.md

```diff
@@ -0,0 +1,158 @@
+# Prometheus Governance v1.0
+
+This document describes the rules and governance of the project. It is meant to be followed by all the developers of the project and the Prometheus community. Common terminology used in this governance document are listed below:
+
+* **Team members**: Any members of the private [prometheus-team][team] Google group.
+
+* **Maintainers**: Team members who are leads on an individual project ([MAINTAINERS.md][maintainers.md]).
+
+* **Projects**: Any repository in the Prometheus [GitHub organization][gh].
+
+* **The Prometheus Project**: The sum of all activities performed under this governance, concerning one or more Repositories or the community.
+
+## Values
+
+The Prometheus developers and community are expected to follow the values defined in the [CNCF charter][charter], including the [CNCF Code of Conduct][coc]. Furthermore, the Prometheus community strives for kindness, giving feedback effectively and building a welcoming environment. The Prometheus developer community strives to operate by rough consensus for the majority of decisions and only resort to conflict resolution through a motion if consensus cannot be reached.
+
+## Projects
+
+Each project must have a MAINTAINERS.md file with at least one maintainer. Where a project has a release process, access and documentation should be such that more than one person can perform a release. Releases should be announced on the prometheus-users mailing list. Any new repositories should be first proposed on prometheus-team@ and following the voting procedures listed below. When a repository is no longer relevant, it should be moved to the prometheus-junkyard Github organization.
+
+## Decisionmaking
+
+### Team members 
+
+Team member status is given to those who have made ongoing contributions for at least 3 months to the Prometheus Project. This is usually in the form of code improvements and/or notable work on documentation, but organizing events or user support could also be taken into account. 
+
+New members may be proposed by any existing member by email to [prometheus-team][team] and voted on by [consensus](#consensus). If no consensus is reached, the new member is not accepted.
+
+Team members are added to the [GitHub organization][gh] and have commit rights to all projects. They should however respect the maintainers of each project.
```

Comment:
My understanding was that we have consensus on members == team == github org. Please speak up if you think not. We can fix the github org once this passes.

Reply:
Not sure if it makes sense to make these exactly equivalent. We might have prometheus-team members that mainly do community stuff, or we might have GH org members that are there mainly for administrative purposes (like Chris A.). I think we should just give out GH permissions on an as-needed basis.

Reply:
We had that consensus a year ago, I think it's fine to state as a principle even if practice varies a little.

Reply:
We strongly favor technical contribution as a reason for team membership, so I think in the regular case there is a need. I'd rather not introduce another step / privilege level tbh, and I don't think it would be worth the effort. The way it's written it also does not preclude _other_ members if that's necessary for administration. Aside from that, _is_ Chris a member or not under this governance? He's on the mailing list, so if not we'd need to introduce a whole new "observer" class.

Reply:
Ok.

Btw. Chris is not on the prometheus-team list and not a voting Prometheus team member.

Reply:
Ah I misremembered that then. Nevermind!